imagevector/images.yaml

```diff
@@ -174,7 +174,7 @@ images:
       teamname: 'gardener/monitoring-maintainers'
 - name: configmap-reloader
   sourceRepository: github.com/prometheus-operator/prometheus-operator
-  repository: ghcr.io/prometheus-operator/prometheus-config-reloader
+  repository: eu.gcr.io/gardener-project/3rd/prometheus-operator/prometheus-config-reloader
```

Comment:
prometheus-config-reloader will be tackled by https://github.com/gardener/gardener/pull/8751. You could revert.